.github/workflows/static.yml

```diff
@@ -183,7 +183,7 @@ jobs:
       - name: "Migrate database to version x"
         run: |
           cd $GITHUB_WORKSPACE/utilities/db-migration
-          go run main.go migrate_to 11
+          go run main.go migrate_to 12
```

Comment:
Yes, we still need to increment this with every version update. I hope to propose a better solution for this in the not to distant future.